spec/uncruft/deprecate_attribute_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'spec_helper'
+
+RSpec.describe Uncruft::DeprecateAttribute do
+  let(:my_name) { "Jess" }
+
+  subject { klass.new }
+
+  describe '#deprecate_attribute' do
```

Comment:
Nit: `deprecate_attribute` is a class method, so `.deprecate_attribute` would be a more idiomatic RSpec describe